pkg/apis/migration/v1alpha1/directvolumemigration_types.go

```diff
@@ -116,8 +133,10 @@ type DirectVolumeMigrationList struct {
 
 type PodProgress struct {
 	*kapi.ObjectReference       `json:",inline"`
-	LastObservedProgressPercent string `json:"lastObservedProgressPercent,omitempty"`
-	LastObservedTransferRate    string `json:"lastObservedTransferRate,omitempty"`
+	PVCReference                *kapi.ObjectReference `json:"pvcRef,omitempty"`
```

Comment:
@alaypatel07 @djwhatle Just wanted to bring to your attention this API change. This is required for the progress reporting system to show PVC name & total elapsed time in the pipeline progress messages.

Reply:
@pranavgaikwad thanks for the explanation. I am not super proud of introducing the PodProgress type. Retrospecting, it feels like had it been `PVCProgress` we would have had a much cleaner API.